backend/auction/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.contrib import admin
 from django.urls import include, path
 
-from .views import AuctionDetailApiView, AuctionListApiView
+from .views import AuctionDetailApiView, AuctionListApiView, AddToAuctionApiView
 
 urlpatterns = [
     path("", AuctionListApiView.as_view(), name="auction_list"),
     path("<uuid:auction_id>/", AuctionDetailApiView.as_view(), name="auction_detail"),
+    path("add_to_auction/", AddToAuctionApiView.as_view(), name="add_to_auction"),
```

Comment:
Convention is to call the endpoint `<uuid:auction_id>/vehicles/<uuid:vehicle_id>`